dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java

```diff
@@ -813,25 +956,25 @@ protected Object call() throws Exception {
                                 deepStorageBrowserPresenter.logText("Successfully deleted bucket", LogType.SUCCESS);
                             });
 
-                        } catch (final IOException e) {
-                            if (e instanceof FailedRequestException) {
-                                LOG.error("Failed to delete Buckets", e);
-                                Platform.runLater(() -> deepStorageBrowserPresenter.logText("Failed to delete Bucket : " + ((FailedRequestException) e).getError().getMessage(), LogType.ERROR));
-                                ALERT.setContentText("Failed to delete bucket");
-                                ALERT.showAndWait();
-                            } else {
-                                LOG.error("Failed to delete Bucket", e);
-                                Platform.runLater(() -> deepStorageBrowserPresenter.logText("Failed to delete Bucket." + e, LogType.ERROR));
-                                ALERT.setContentText("Failed to delete a bucket");
-                                ALERT.showAndWait();
-                            }
+                        } catch (final FailedRequestException fre) {
```

Comment:
If this catch statement is no different that the IOException catch bellow it, remove this catch